pages/help/glossary.mdx

```diff
@@ -0,0 +1,91 @@
+## General 
+
+| Acronym/Abbreviation         | Definition                                                                               |
+| ---------------------------- | ---------------------------------------------------------------------------------------- |
+| ACL                          | Access Control List                                                                      |
+| API                          | Application Programming Interface                                                        |
+| ARPA                         |  Accountable, Responsible, Participant, Advisor                                          |
+| BAM                          | Bidirectional Associative Memory                                                         |
+| BCR                          | Biospecimen Core Resource                                                                |
+| CI/CD                        |                                                                                          |
+| CNV                          |  Copy Number Variation                                                                   |
+| DCA                          | Data Curator App                                                                         |
+| DCC                          | Data Coordinating Center                                                                 |
+| eRA                          | Electronic Research Administration                                                       |
+| FISH                         |                                                                                          |
+| GCA                          | Genomic Data Commons                                                                     |
+| HTTP                         | Hypertext Transfer Protocol                                                              |
+| IDE                          | Integrated Development Environment                                                       |
```

Comment:
IDE is probably useful for an internal glossary but not an external user-facing one since I don't think I've used this in conversation with someone external before (as opposed to something like REST), so we can probably remove it.